wrapanapi/ec2.py

```diff
@@ -237,6 +250,16 @@ def stack_exist(self, stack_name):
         else:
             return bool(stacks)
 
+    def list_stacks(self, stack_status_filter=stack_states['active']):
+        """
+        Returns a list of Stack objects
+        stack_status_filter:  filters stacks in certain status. check EC2System:stack_states for
```

Comment:
**REQUIRED:**  Are the only valid values the keys of stack_states, or can you for instance filter on ROLLBACK_COMPLETE?   Whatever the answer make that clear in the comment.   It's not now.

Reply:
any value or list of value can be used. I just tried to have same filters that EC2 Web console has.and default I set to 'active' as Deleted stacks leave forever on the portal

Reply:
I mean for the variable stack_status_filter what values are valid?   Is it just a list of stack status codes?   Those codes being defined here under the heading "Stack Status Codes":  

    https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-describing-stacks.html

Reply:
exactly. one can select any of those valid "Stack Status Codes" or list of them. that's why I added some lists for reference

Reply:
OK, so the comment should probably be:

    stack_status_filter:  filters stacks in certain state.   Can be a either a single valid stack status code or 
    or a list of them.  Check EC2System:stack_states for details.